project/model.py

```diff
@@ -0,0 +1,78 @@
+from torch.nn import Linear, Tanh, LSTM, Embedding
+from torch.optim import SGD
+import pytorch_lightning as pl 
+from torchvision.models import resnet18
+from loss_func.simclr import SimCLR
+import torch
+
+class JeopardyModel(pl.LightningModule):
+    def __init__(self, vocab_sz, bs):
```

Comment:
don't put bs as an argument, we generaly want to refrain from hardcoding to a bs

Reply:
so the only place I use the batch size is when I initialize the LSTM hidden state and cell state with all zeroes - and that's why I need this argument. I'll think of possible workarounds